src/graphql-samples/mutations/update/update-tags-on-tagset

```diff
@@ -0,0 +1,19 @@
+mmutation updateTagsOnTagset($tagsInput: TagsInput!, $tagsetID: Float!) {
```

Comment:
the example mutation could be re-written to 

```
mutation {
  updateTagsOnTagset(
    tags: {
      tags: ["tag1", "tag2"]
    }
    tagsetID: 4
  )
  {
    name
    tags
  }
}
```

for easy execution in GraphQL playground